google-cloud-apis/java-apis/maven-apis/src/com/google/cloud/tools/intellij/cloudapis/maven/CloudLibraryDependencyWriter.java

```diff
@@ -101,44 +103,55 @@ private static void addLibrariesToMavenModule(
       @NotNull Set<CloudLibrary> libraries, @NotNull Module module, @Nullable String bomVersion) {
     Project project = module.getProject();
     MavenProject mavenProject = MavenProjectsManager.getInstance(project).findProject(module);
-    MavenDomProjectModel model =
-        MavenDomUtil.getMavenDomProjectModel(project, mavenProject.getFile());
+    MavenDomProjectModel model;
+    if (mavenProject != null) {
+      model = MavenDomUtil.getMavenDomProjectModel(project, mavenProject.getFile());
```

Comment:
I assume the result of this call never can be `null` ( I mean `getMavenDomProjectModel`)?

Reply:
it is actually `@Nullable`. I added a check for this too.